src/evaluate.jl

```diff
@@ -141,11 +137,10 @@ function evaluate!(x::Array{TaylorN{T},1}, δx::Array{Taylor1{T},1},
 end
 
 function evaluate!(x::Array{TaylorN{T},1}, δx::Array{TaylorN{T},1},
-        x0::Array{TaylorN{T},1}) where {T<:NumberNotSeriesN}
+        x0::Array{TaylorN{T},1}; sorting::Bool=true) where {T<:NumberNotSeriesN}
 
-    # @assert length(x) == length(x0)
     @inbounds for i in eachindex(x, x0)
-        x0[i] = evaluate( x[i], δx )
+        x0[i] = _evaluate( x[i], δx, Val(sorting) )
```

Comment:
I don't think you gain anything by using `Val` here.

Reply:
I guess that the gain is dispatch rather than an `if`.